src/index.js

```diff
@@ -670,7 +671,18 @@ const members = {
       return this.getAttribute('name');
     },
     set(name) {
-      return this.setAttribute('name', name);
+      const ret = this.__setAttribute('name', name);
```

Comment:
Apparently :) Or there we troubles with Travis that looked exactly like AtlasKit limit problems.